conversions/regex.inc

```diff
@@ -41,11 +43,51 @@ function coder_upgrade_upgrade_regex_alter(&$file) {
   $from[] = "/format_username/";
   $to[] = "user_format_name";
 
+  // theme_html_tag() has been renamed.
+  $from[] = "/theme_html_tag/";
+  $to[] = "theme_head_tag";
+
+  // hook_page_build() has been replaced: https://api.backdropcms.org/change-records/new-layout-module-implements-drag-drop-model-building-layouts
+  $from[] = "/".$name."_page_build/";
+  $to[] = $name."_preprocess_page";
+
+  // hook_library() has been renamed.
+  $from[] = "/".$name."_library/";
+  $to[] = $name."_library_info";
+
+  // hook_library_alter() has been renamed.
+  $from[] = "/".$name."_library_alter/";
+  $to[] = $name."_library_info_alter";
+
+  // replace special sorting function: https://api.backdropcms.org/change-records/generic-sorting-function-replace-implementations-uasort
+  $from[] = '/uasort\(\$items, *'."'element_sort'\)/";
+  $to[] = 'backdrop_sort($items, array("#weight"))';
+
+  // replace special sorting function 2
+  $from[] = '/uasort\(\$items, *'."'element_sort_title'\)/";
+  $to[] = 'backdrop_sort($items, array("#title" => SORT_STRING))';
+
+  // replace special sorting function 3
+  $from[] = '/uasort\(\$items, *'."'drupal_sort'\)/";
+  $to[] = 'backdrop_sort($items)';
+
+  // replace special sorting function 4
+  $from[] = '/uasort\(\$items, *'."'drupal_sort_title'\)/";
+  $to[] = 'backdrop_sort($items, array("title" => SORT_STRING))';
+
+  // replace special sorting function 5
+  $from[] = '/uasort\(\$items, *'."'admin_bar_element_sort'\)/";
+  $to[] = 'backdrop_sort($items, array("#weight" => SORT_NUMERIC, "#title" => SORT_STRING))';
+
   // temporarily rename Drupal links so the other regexes dont create false 
   // links.
   $from[] = "/drupal\.org/";
   $to[] = "mothershiporg";
 
+  // change 'Drupal 7' to 'Backdrop CMS' throughtout.
```

Comment:
This change should not be automated. Consider the following sentence for example:

> Backdrop CMS does things differently than Drupal 7.

...would become:

> Backdrop CMS does things differently than Backdrop CMS.